tensorflow_quantum/python/layers/circuit_executors/expectation_test.py

```diff
@@ -192,28 +275,41 @@ def test_expectation_simple_tf_train(self):
         self.assertAllClose(mse.numpy(), 0, atol=1e-3)
 
 
-class ExpectationFunctionalTests(tf.test.TestCase):
+class ExpectationFunctionalTests(parameterized.TestCase, tf.test.TestCase):
     """Test hybrid/integrated models that include an expectation layer."""
 
-    def test_simple_param_value_input(self):
+    @parameterized.parameters([
+        {
+            'backend': 'noisy'
+        },
+        {
+            'backend': None  # old API usage
+        }
+    ])
+    def test_simple_param_value_input(self, backend):
         """Train a densely connected hybrid model.
 
         This model will put a qubit in the zero or one state from a random state
         given the input zero or one. This tests the input signature:
         Expectation([input_value_batch]).
         """
+        noisy = backend == 'noisy'
         bit = cirq.GridQubit(0, 0)
         symbols = sympy.symbols('x y z')
-        circuit = _gen_single_bit_rotation_problem(bit, symbols)
+        circuit = _gen_single_bit_rotation_problem(bit, symbols,
+                                                   True if noisy else False)
```

Comment:
can just be `noisy` instead of `True if noisy else False` ?

Reply:
It totally can be lol